Changes

```diff
@@ -258,6 +258,9 @@ Working version
    Guillaume Munch-Maccagnoni, KC Sivaramakrishnan, Stefan Muenzel,
    Xavier Leroy)
 
+- #12307: Add BLAKE2b hashing and an MD5 submodule to the Digest module.
+  (Xavier Leroy, review by Olivier Nicole)
```

Comment:
(I suspect that more reviewers deserve a mention: myself, wikku (Wiktor Kuchta), Daniel and, possibly Hannes.)

Reply:
I agree with @xavierleroy.